stable/redis-ha/values.yaml

```diff
@@ -91,6 +91,22 @@ affinity: |
               release: {{ .Release.Name }}
           topologyKey: failure-domain.beta.kubernetes.io/zone
 
+# Prometheus exporter specific configuration options
+exporter:
+  enabled: false
+  image: oliver006/redis_exporter
+  tag: v0.28.0
+  pullPolicy: IfNotPresent
+
+  # Annotations for prometheus
+  annotations:
+    prometheus.io/port: "9121"
```

Comment:
Would be good to have the port configurable under `.Values.exporter.port` as well.

Reply:
Done - actually, I made both the port and the scrape path configurable